ioctl/newcmd/account/accountcreate.go

```diff
@@ -76,14 +79,11 @@ func NewAccountCreate(c ioctl.Client) *cobra.Command {
 			}
 
 			message := createMessage{Accounts: newAccounts}
-
-			fmt.Println(message.String())
+			c.PrintInfo(message.String())
```

Comment:
can we directly use `output.PrintResult(message.String())`?

Reply:
the same above